src/circuits/runcircuit.jl

```diff
@@ -248,53 +203,138 @@ The starting state is generated automatically based on the flags `process`, `noi
    `Λ = ε⊗1̂(|ξ⟩⟨ξ|)`, where `|ξ⟩= ⨂ⱼ |00⟩ⱼ+|11⟩ⱼ`, approximated by a MPO with 4 site indices,
    two for the input and two for the output Hilbert space of the quantum channel.
 """
-function runcircuit(sites::Vector{<:Index}, circuit::Union{Tuple,Vector{<:Any},Vector{Vector{<:Any}}};
-                    process = false, noise = nothing, kwargs...)
+function runcircuit(sites::Vector{<:Index}, 
+                circuit::Union{Tuple,Vector{<:Any},Vector{Vector{<:Any}}};
+                process::Bool = false, noise = nothing,
+                exact::Bool = false,
```

Comment:
I'm going back and forth about this, but maybe `full_state` is a better name since it gives a better idea of how it is implemented (there are many different ways of doing an "exact" simulation, including MPS without truncation). The only issue there is the process/Choi case, but I see that Choi matrix can also be referred to as a Choi state so it could be ok. Or maybe `full_representation`?